WORKSPACE

```diff
@@ -1,5 +1,25 @@
 workspace(name = "rules_bzlmodrio_toolchains")
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_cc",
```

Comment:
I'm not really sure why we have `rules_cc` both in the WORKSPACE and bzlmod.

Reply:
buildifier is adding a rules_cc load to one of the files.  The load that it adds only exists in newer rules_cc.  Kinda annoying, but these things are generally not worth fighting in the long run.